doc/dev/crimson/crimson.rst

```diff
@@ -144,86 +122,36 @@ one of the two following configuration options:
 Running Crimson
 ===============
 
-As you might expect, Crimson does not yet have as extensive a feature set as does ``ceph-osd``.
-
-object store backend
---------------------
-
-At the moment, ``crimson-osd`` offers both native and alienized object store
-backends. The native object store backends perform IO using the SeaStar reactor.
-They are:
-
-.. describe:: cyanstore
-
-   CyanStore is modeled after memstore in the classic OSD.
-
-.. describe:: seastore
-
-   Seastore is still under active development.
-
-The alienized object store backends are backed by a thread pool, which
-is a proxy of the alienstore adaptor running in Seastar. The proxy issues
-requests to object stores running in alien threads, i.e., worker threads not
-managed by the Seastar framework. They are:
-
-.. describe:: memstore
-
-   The memory backed object store
-
-.. describe:: bluestore
-
-   The object store used by the classic ``ceph-osd``
-
-daemonize
----------
+.. note::
 
-Unlike ``ceph-osd``, ``crimson-osd`` does not daemonize itself even if the
-``daemonize`` option is enabled. In order to read this option, ``crimson-osd``
-needs to ready its config sharded service, but this sharded service lives
-in the Seastar reactor. If we fork a child process and exit the parent after
-starting the Seastar engine, that will leave us with a single thread which is
-a replica of the thread that called `fork()`_. Tackling this problem in Crimson
-would unnecessarily complicate the code.
+   #. Crimson is in a tech preview stage. 
+   #. As you might expect, Crimson does not yet have as extensive a feature set as does ceph-osd. 
+         Malfunctions including crashes and data loss are to be expected. 
```

Comment:
nit: Indentation is not accurate here. You can find the rendered docs here: https://ceph--58705.org.readthedocs.build/en/58705/dev/crimson/crimson/#running-crimson

```
.. note::
    Crimson is in a tech preview stage. 
    As you might expect, Crimson does not yet have as extensive a feature set as does ceph-osd. 
    Malfunctions including crashes and data loss are to be expected. 
```